### PR TITLE
Remove inactive exchanges

### PR DESCRIPTION
**Pull request by qertoip**
Not affiliated with Changelly, just cleaning up the list.

### CODE REVIEW

**Comment by mattcode55** [comment]
-1, Changelly is listed as "untrusted by the community" in the sidebar of [/r/monero](https://www.reddit.com/r/Monero/).

**Comment by qertoip** [comment]
Thanks @mattcode55. I removed the Changelly.

File: _data/merchants.yml [hunk 1]
```diff
@@ -32,14 +30,13 @@
...
-    - name: BIT.AC
-      url: https://bit.ac/
     - name: Kaiserex
       url: https://www.kaiserex.com/
     - name: Magnetic Exchange
       url: https://magneticexchange.com
     - name: ArbitrCoin
-      url: https://arbitrcoin.com      
+      url: https://arbitrcoin.com 
+
```
Comment (MaxXor) [inline _data/merchants.yml]: Please remove this empty line. Everything else looks good.

Comment (qertoip) [inline _data/merchants.yml]: The empty line is now removed, thanks!

**Comment by fluffypony** [comment]
Please rebase

**Comment by qertoip** [comment]
Rebased.

**Pull request merged**